CI/ios/before_install.sh

```diff
@@ -3,5 +3,5 @@
 echo DEVELOPER_DIR=/Applications/Xcode_14.2.app >> $GITHUB_ENV
 
 mkdir ~/.conan ; cd ~/.conan
-curl -L 'https://github.com/vcmi/vcmi-ios-deps/releases/download/1.2.1/ios-arm64.txz' \
-	| tar -xf -
+curl -L 'https://github.com/vcmi/vcmi-dependencies/releases/download/1.2/dependencies-ios.tgz' \
+	| tar -xzf -
```

Comment:
-z not needed on macOS in x mode

Reply:
I actually wanted to unify this code in a single .sh file.

Right now we have like 7 `before_install.sh`, all of which execute mkdir, cd, curl & tar. And you need to change url in each of them when updating dependencies, which is a pain.

So I would rather create shared `install_dependencies.sh` file that will be executed for all conan CI's

Reply:
single script makes complete sense, I fully agree